ert_gui/tools/plot/plot_api.py

```diff
@@ -209,12 +205,13 @@ def _add_index_range(self, data):
         index = pd.MultiIndex.from_tuples(tuples, names=["key_index", "data_index"])
         data.columns = index
 
-    def refcase_data(self, key):
-        """Returns a pandas DataFrame with the data points for the refcase for a given data key, if any.
-        The row index is the index/date and the column index is the key."""
-        return self._facade.refcase_data(key)
-
     def history_data(self, key, case=None):
         """Returns a pandas DataFrame with the data points for the history for a given data key, if any.
         The row index is the index/date and the column index is the key."""
-        return self._facade.history_data(key, case)
+
+        if ":" in key:
+            head, tail = key.split(":", 2)
+            history_key = "{}H:{}".format(head, tail)
```

Comment:
Pylint will complain here if you don't use `f-strings`.

Reply:
might be - just moved old code from one function to another here - haven't made this myself. I will have a look at it :)